### PR TITLE
add zendesk, intercom, and wordpress API loaders

### PR DESCRIPTION
**Pull request by bborn**
@jerryjliu  here's a PR for Zendesk, Intercom, and Wordpress loaders.

### CODE REVIEW

File: loader_hub/intercom/base.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+"""Intercom reader."""
+from typing import List
+import requests
+import json
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from bs4 import BeautifulSoup
```
Comment (jerryjliu) [inline loader_hub/intercom/base.py:7]: you may have to do lazy importing of bs4 for tests to pass. see `loader_hub/web/beautiful_soup_web/base.py` as an example 

(i know it's pretty hacky)

File: loader_hub/wordpress/base.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+"""Wordpress reader."""
+from typing import List
+import json
+import requests
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from bs4 import BeautifulSoup
```
Comment (jerryjliu) [inline loader_hub/wordpress/base.py:7]: same here

File: loader_hub/zendesk/base.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
+"""Zendesk reader."""
+from typing import List
+import json
+import requests
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from bs4 import BeautifulSoup
```
Comment (jerryjliu) [inline loader_hub/zendesk/base.py:7]: same here

**Review by jerryjliu — CHANGES_REQUESTED** [review]
thanks for adding these! 

note: you'll want to add to library.json as well (and the comments about lazy importing will be more relevant then)

**Comment by bborn** [comment]
@jerryjliu ok, I *think* I did what you wanted. I can't run the tests locally for some reason (I get `E   KeyError: PosixPath('/Users/bruno/Projects/python/llama-hub')`) which I don't understand

**Review by jerryjliu — APPROVED** [review]
looks good!

> @jerryjliu ok, I _think_ I did what you wanted. I can't run the tests locally for some reason (I get `E KeyError: PosixPath('/Users/bruno/Projects/python/llama-hub')`) which I don't understand

are you just doing `pytest tests` from within your llama-hub fork? i cloned your repo and it's working for me

**Pull request merged**